pymess/models/emails.py

```diff
@@ -61,6 +61,9 @@ def friendly_sender(self):
     def failed(self):
         return self.state == self.STATE.ERROR
 
+    def __str__(self):
+        return str(self.recipient)
```

Comment:
I would add more information to string representation of the email message. For example:
`recipient@gmail.com: Subject of the message`.

Reply:
This way it's consistent with [sms model](https://github.com/druids/django-pymess/blob/master/pymess/models/sms.py#L69)

Reply:
Well.... ok, but just recipient is IMO not descriptive enought for the whole object.

Reply:
Agree, maybe both models should be fixed. Let's master druid resolves it.